index.html

```diff
@@ -9,6 +9,8 @@
 
 <body>
     <h1 style="color:red">This is my welcome page</h1>
+
+    <footer style="color:grey">footer by zunyang</footer>
```

Comment:
Add your student id in the footer and make it bold.

Reply:
Sure, can do.